Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/node:20.11.1-alpine
+FROM docker.io/node:lts-alpine3.20
```

Comment:
Isn't it generally a better idea to use a specific version number instead of just the `lts` tag? Things could break in a future LTS version.

Reply:
Oh i think youre right.
we could use [20.15.1-alpine3.20](https://hub.docker.com/layers/library/node/20.15.1-alpine3.20/images/sha256-aa22e2c263d7242e7452d1efa40a031ac8ccf2305cc4d925c737486d339bd178?context=explore) instead